docs/eventing/sources/README.md

```diff
@@ -1,23 +1,44 @@
-An event source is a resource object that acts as a link between an event producer and an event _sink_. A sink can be a Knative Service, Channel, or Broker that receives events from an event source.
+An event source is a Kubernetes custom resource (CR), created by a developer or cluster administrator, that acts as a link between an event producer and an event _sink_.
+A sink can be a k8s service, including Knative Services, a Channel, or a Broker that receives events from an event source.
 
-This page provides an index of the available source resource types as well as
-links to their documentation.
+Event sources are created by instantiating a CR from a Source object.
+The Source object defines the arguments and parameters needed to instantiate a CR.
+
+All Sources are part of the `sources` category.
+
+{{< tabs name="List event sources" default="kubectl" >}} {{% tab name="kubectl" %}}
```

Comment:
shortcodes cannot be used in README.md because `readfile` (see `_index.md`) does not process shortcodes.